modules/cnv-vm-wizard-fields-web.adoc

```diff
@@ -42,37 +34,43 @@ endif::[]
 |Container
 |Provision virtual machine from a bootable operating system container located in a registry accessible from the cluster. Example: `_kubevirt/cirros-registry-disk-demo_`.
 
-|Cloned Disk
-|Provision source is a cloned disk.
+|Disk
+|Provision virtual machine from a disk.
 
-|Import
-|Import virtual machine from a supported provider.
+|Attach disk
+|
+|Attach an existing disk that was previously cloned or created and made available in the Persistent Volume Claims. When this option is selected, you must manually complete the *Operating System*, *Flavor*, and *Workload Profile*.
```

Comment:
s/Persistent Volume Claims/PersistentVolumeClaims

Reply:
Changed to: PersistentVolumeClaims